### DISCUSSION TITLE
Feature Request: Add a "Force authentication for approval" in branch protection rules

### DISCUSSION THREAD

**Discussion by mkerollos** [community/community#7974] [Pull Requests]
Medical devices in the US are regulated by the Code of Federal Regulations (CFR) Title 21. All records (which includes release candidates, design documents, tests, etc.) must comply with [CFR Title 21 Part 11 ](https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfcfr/CFRSearch.cfm?CFRPart=11&showFR=1&subpartNode=21:1.0.1.1.9.3) part of which mandates that approvals of records must be performed using a password to ensure that the approver is who they say they are.

This feature request is to add an option to branch protection rules to enforce a user to authenticate when they are providing an approving vote on a PR.

This is a feature that exists natively in [gitlab](https://gitlab.com/gitlab-org/gitlab/-/merge_requests/10364) and through the marketplace on [bitbucket server](https://marketplace.atlassian.com/apps/1211303/workzone-pullrequest-workflow?hosting=server&tab=overview)

**Comment by rebsp** [comment]
this is something we are missing in our org as well and should be fairly straight-forward to implement imho. although reasonably popular no feedback from GitHub here in over a year, care to take a look @davidstaheli or someone else from product management? cheers
